### DISCUSSION TITLE
how to know whether i am administrator of one Wechat group?

### DISCUSSION THREAD

**Discussion by tomallv** [wechaty/wechaty#2062] [Q&A] [answered]
I want to know whether i am the administrator in the room，and have been told that can get admin info from room.payload by some guys. but i found that room.payload.adminIdList is still empty although the room has been set one administrator after try. i don't know whether this is a bug or can not use this way? thanks!

**Comment by qiaojianfeng** [comment]
这个问题解决了吗？

**Comment by huan** [comment, marked as answer]
Whether the Wechaty can know the admin of a room, is highly dependent on the puppet that you are using.

For example, the wechaty-puppet-puppeteer can not get to know who is the admin of a room because the Web Protocol does not support that, but the wechaty-puppet-donut (Win32) or wechaty-puppet-padlocal (Pad) might have this information.

**Comment by qiaojianfeng** [comment]
> 这个问题解决了吗？



> Whether the Wechaty can know the admin of a room, is highly dependent on the puppet that you are using.
> 
> For example, the wechaty-puppet-puppeteer can not get to know who is the admin of a room because the Web Protocol does not support that, but the wechaty-puppet-donut (Win32) or wechaty-puppet-padlocal (Pad) might have this information.

谢谢您的回复。我使用的是 wechaty-puppet-padplus（0.7.42），我想确认一下这个协议可以是否可以获取群管理数据

**Comment by huan** [comment]
@webweifeng I believe @windmemory @su-chang is the right people who can answer your question because they are the maintainers of the wechaty-puppet-padplus.
